datafusion/expr/src/aggregate_function.rs

```diff
@@ -77,30 +77,34 @@ impl FromStr for AggregateFunction {
     type Err = DataFusionError;
     fn from_str(name: &str) -> Result<AggregateFunction> {
         Ok(match name {
-            "min" => AggregateFunction::Min,
-            "max" => AggregateFunction::Max,
-            "count" => AggregateFunction::Count,
+            // general
```

Comment:
just reorganizing, to be similar to how it looks in the sql doc, and group them together here a bit better